config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -115,3 +115,53 @@ periodics:
           # this is mostly for building kubernetes
           memory: 9Gi
           cpu: 7
+- interval: 6h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-compatibility-versions-feature-gate-test
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: compatibility-versions-feature-gate-test
+    description: Uses kind to run bespoke feature gate tests from the n-1 kubernetes release yaml files against a latest kubernetes master components w/ --emulated-version=n-1 set.
+    # TODO(aaron-prindle) route the alert email to a rotation vs individual email
+    testgrid-alert-email: aprindle@google.com
+    testgrid-num-columns-recent: '6'
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+    workdir: true
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/krte:v20241230-3006692a6f-master
+      imagePullPolicy: Always  # pull latest image for canary testing
+      command:
+      - wrapper.sh
+      - bash
+      - -c
+      - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && ./../test-infra/experiment/compatibility-versions/compatibility-versions-feature-gate-test.sh
```

Comment:
xref: https://github.com/kubernetes/test-infra/issues/33980

Reply:
Apologies is the recommendation here related to: 
https://github.com/kubernetes/test-infra/issues/33594

From reading over https://github.com/kubernetes/test-infra/issues/33980 I'm a bit confused what the actionable change would be to make to the script here is, can you elaborate a bit on how I might use the precompiled binaries here?  Thanks!

Reply:
Sorry -- I'm just trying to track these for later, it's not a blocker.

Basically you use `kind build node-image` with one of the CI builds of Kubernetes instead of from source.
We will also have to make sure the commit from that build gets recorded to testgrid (I forget the specifics of how that works), if / when we convert one of the scripts we can apply it in multiple places.

But it's not a blocker.

Reply:
Ah makes sense, thanks.  I'll track this work through https://github.com/kubernetes/test-infra/issues/33594, added a reference there to https://github.com/kubernetes/test-infra/issues/33980.  I'll work on a separate PR converting all of the compatibility-versions pieces to use prebuilt binaries when I tackle https://github.com/kubernetes/test-infra/issues/33594